code.py

```diff
@@ -1,106 +1,91 @@
-# Reference Software
-# By Joseph R. Freeston
-
-from time import sleep
-import board
-import struct
-import pulseio
-import digitalio
 import adafruit_bmp280
+import alarm
+import board
 import time
+import traceback
+
 import servercom
-from reference_client import ReferenceClient
-from microcontroller import watchdog as w
-from watchdog import WatchDogMode
-import indication
-import supervisor
 
-import reference_config
+
+@contextmanager
+def connection():
+	c = servercom.Connection()
+	try:
+		yield c
+	finally:
+		c.close()
+
```

Comment:
[Like in the beacon code](https://github.com/Windham-High-School/CubeServer-beacon/pull/2#discussion_r1208065542), CircuitPython does not implement this.

```suggestion
connection = servercom.Connection
```

Reply:
import fixed.